service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationController.java

```diff
@@ -43,4 +50,15 @@ public ResponseEntity createdCase(
 
         return ResponseEntity.ok(body);
     }
+
+    @PostMapping("/submitted")
+    public ResponseEntity handleCaseInitiation(
```

Comment:
Please rename method to be something like `handleSubmittedEvent`.

Please also rename `createdCase` to something like `handleAboutToSubmitEvent`